cmake/OpenCVPackaging.cmake

```diff
@@ -246,6 +246,10 @@ endif()
 set(DEV_LINTIAN_OVERRIDES "binary-or-shlib-defines-rpath" # usr/bin/opencv_traincascade
                           "binary-without-manpage") # usr/bin/opencv_traincascade
 
+if(AARCH64)
```

Comment:
The problem origin is not architecture, but lintian tool version. Ubuntu 15.04 includes newer version of lintian that supports more directives.